libs/basics/src/assert.test.ts

```diff
@@ -90,4 +91,5 @@ test('assertFalsy', () => {
   expect(() => assertFalsy(truthyValue)).toThrowError(
     `Unexpected truthy value: ${truthyValue}`
   );
+  expect(() => assertFalsy(0)).not.toThrowError();
```

Comment:
This was flagged as a case that wasn't covered. Different `istanbul` version from what `jest` had, I guess?